lib/RESTClient.js

```diff
@@ -393,6 +395,53 @@ class RESTClient {
             callback(errors.InternalError);
         }).end();
     }
+
+    /**
+    *   send a request to get all raft sessions
+    *   get server's response and return it
+    *
+    *   @param {string} reqUids - the identifier of the request
+    *   @param {callback} callback - callback(err, info)
+    *   @param {werelogs.Logger} [reqLogger] - Logger instance
+    *   @return {undefined}
+    */
+    getAllRafts(reqUids, callback, reqLogger) {
+        const log = reqLogger || this.createLogger(reqUids);
+        const path = '/_/raft_sessions/';
+        log.debug('getting all raftSessions');
+        this.request('GET', path, log, null, null, callback);
+    }
+
+    /**
+    *   Get raft logs from bucketd
+    *   get server's response and return it
+    *
+    *   @param {string} raftId - raft session id
+    *   @param {number} [start=1] - starting sequence number
+    *   @param {number} [end=10000] - ending sequence number
+    *   @param {boolean} [targetLeader=false] - true: from leader instead of
+    *       follower
+    *   @param {string} reqUids - the identifier of the request
+    *   @param {callback} callback - callback(err, info)
+    *   @param {werelogs.Logger} [reqLogger] - Logger instance
+    *   @return {undefined}
+    */
+    getRaftLog(raftId, start, end, targetLeader, reqUids, callback, reqLogger) {
+        const log = reqLogger || this.createLogger(reqUids);
+        const path = `/_/raft_sessions/${raftId}/log`;
+        const query = {};
+        if (start !== undefined) {
+            query.start_sequence = start;           // eslint-disable-line
+        }
+        if (end !== undefined) {
+            query.end = end;
+        }
+        if (targetLeader !== undefined) {
```

Comment:
Check for undefined or null here, to make sure that we handle "non-provided values" from the API user.